x/uibc/quota/keeper/genesis_test.go

```diff
@@ -0,0 +1,41 @@
+//go:build experimental
+// +build experimental
+
+package keeper_test
+
+import (
+	"testing"
+
+	"github.com/umee-network/umee/v4/x/uibc"
+	"gotest.tools/v3/assert"
+)
+
+func TestInitGenesis(t *testing.T) {
```

Comment:
let's separate unit tests from integration tests - to make sure we can run only unit tests.